modules/examples/src/main/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementExamples.java

```diff
@@ -0,0 +1,357 @@
+// ENTERPRISE_MANAGEMENT_ENTERPRISE_ID=<ID of the enterprise>
+// ENTERPRISE_MANAGEMENT_ACCOUNT_ID=<enterprise account ID>
+// ENTERPRISE_MANAGEMENT_ACCOUNT_IAM_ID=<IAM ID of the enterprise account>
+//
+// These configuration properties can be exported as environment variables, or stored
+// in a configuration file and then:
+// export IBM_CREDENTIALS_FILE=<name of configuration file>
+//
+public class EnterpriseManagementExamples {
+
+  private static final Logger logger = LoggerFactory.getLogger(EnterpriseManagementExamples.class);
+
+  protected EnterpriseManagementExamples() {
+  }
+
+  static {
+    System.setProperty("IBM_CREDENTIALS_FILE", "../../enterprise_management.env");
+  }
+
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    EnterpriseManagement service = EnterpriseManagement.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> config = CredentialUtils.getServiceProperties(EnterpriseManagement.DEFAULT_SERVICE_NAME);
+
+    String url = config.get("URL");
+    String authType = config.get("AUTHTYPE");
+    String authUrl = config.get("AUTH_URL");
+    String apiKey = config.get("APIKEY");
+    String enterpriseId = config.get("ENTERPRISE_ID");
+    String enterpriseAccountId = config.get("ACCOUNT_ID");
+    String enterpriseAccountIamId = config.get("ACCOUNT_IAM_ID");
+    String firstAccountGroupName = "Example Account Group";
+    String firstAccountGroupId = null;
+    String secondAccountGroupName = "Second Example Account Group";
+    String secondAccountGroupId = null;
+    String firstUpdatedAccountGroupName = "Updated Example Account Group";
+    String accountName = "Example Account Name";
+    String accountId = null;
+    String updatedEnterpriseName = "Updated Enterprise Name";
+
+    try {
+      // begin-create_account_group
+      String parent = "crn:v1:bluemix:public:enterprise::a/%s::enterprise:%s";
+      CreateAccountGroupOptions createAccountGroupOptions = new CreateAccountGroupOptions.Builder()
+          .parent(String.format(parent, enterpriseAccountId, enterpriseId))
+          .name(firstAccountGroupName)
+          .primaryContactIamId(enterpriseAccountIamId)
+          .build();
+
+      Response<CreateAccountGroupResponse> response = service
+          .createAccountGroup(createAccountGroupOptions)
+          .execute();
+      CreateAccountGroupResponse createAccountGroupResponse = response.getResult();
+
+      System.out.println(createAccountGroupResponse);
+      // end-create_account_group
+      firstAccountGroupId = createAccountGroupResponse.getAccountGroupId();
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s%nError details: %s", e.getStatusCode(),
+          e.getMessage(), e.getDebuggingInfo()), e);
+    }
+
+    try {
+      // begin-create_account_group
```

Comment:
Let's remove the begin/end comments for this block since we can only include one code example for each operation.